### PR TITLE
add pitr and dual region doc

### PR DESCRIPTION
**Pull request by IANTHEREAL**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add pitr and dual region doc

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by grovecai** [comment]
LGTM

File: tidb-cloud/backup-and-restore.md [hunk 1]
```diff
@@ -14,25 +14,47 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+    - select **Backup Retention**, which is the minimum backup data retention period.
+
+    - select **Backup Storage Region**, in addition to storing the backup data in where the cluster is located, you can add another remote region, TiDB Cloud will copy all new backup data to the remote region which facilitates data safety and faster recovery.
 
 4. Click **Confirm**.
 
-If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
+> **Note that:**
+> You can not disable automatic backup.
+> To use the PITR feature, firstly make sure your TiDB cluster version is at least v6.3.0 and the TiKV node configuration is at least 8c/16g, then file a ticket to request to enable the PITR feature.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:43]: 这一条提示放到用户**确认 PITR 开启**的步骤（L31）下面吧

File: tidb-cloud/backup-and-restore.md [hunk 2]
```diff
@@ -14,25 +14,47 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+> After adding a remote region as a backup data store, you can't turn it off.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:44]: 同上，这一条提示放到用户**选择 region** 的步骤（L37）下面

File: tidb-cloud/backup-and-restore.md [hunk 3]
```diff
@@ -14,25 +14,47 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
 
 1. Navigate to the **Backup** tab of a cluster.
 
-2. Click **Auto Setting**. The setting window displays.
+2. Click **Backup Settings**. The setting window displays.
 
-3. In the setting window, select **Backup Time** and maximum backup files in **Limit (backup files)**.
+3. In the setting window,
+
+    - check whether the PITR(**Point-in-time Recovery**) feature is on.
```
[the remark this answers is not in the document]
Comment (IANTHEREAL) [reply tidb-cloud/backup-and-restore.md:31]: 最初版本我是写在这里的，但是感觉如果所有限制都写在操作里面内容有点多。 你觉得呢，放在下面影响阅读吗。 

目前这个只能显示状态 On，如果 PITR 打开的话。接下来几个版本会让用户自己打开和关闭

**Comment by IANTHEREAL** [comment]
@ran-huang @grovecai PTAL

File: tidb-cloud/backup-and-restore.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (ran-huang) [reply tidb-cloud/backup-and-restore.md:31]: 我建议是把每一条提示都尽可能放到离操作最近的地方，否则用户不好找到。

File: tidb-cloud/backup-and-restore.md [hunk 4]
```diff
@@ -14,25 +14,47 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
```
Comment (IANTHEREAL) [inline tidb-cloud/backup-and-restore.md:31]:
```suggestion
    - check whether the PITR(**Point-in-time Recovery**) feature is on. To use the PITR feature, firstly make sure your TiDB cluster version is at least v6.3.0 and the TiKV node configuration is at least 8c/16g, then file a ticket to request to enable the PITR feature.
```

File: tidb-cloud/backup-and-restore.md [hunk 5]
```diff
@@ -14,25 +14,47 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+
+    - In **Backup Time**, schedule a start time for the daily cluster backup. It is recommended to schedule automatic backup at the low workload period. If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
+
+    - In **Backup Retention**, configure the minimum backup data retention period.
+
+    - In **Backup Storage Region**, select the regions where you want to store your backup data. TiDB Cloud stores your backup data in your local region by default. In addition, you can add another remote region, and TiDB Cloud will copy all new backup data to the remote region, which facilitates data safety and faster recovery.
```
Comment (IANTHEREAL) [inline tidb-cloud/backup-and-restore.md:37]:
```suggestion
    - In **Backup Storage Region**, select the regions where you want to store your backup data. TiDB Cloud stores your backup data in your local region by default. In addition, you can add another remote region, and TiDB Cloud will copy all new backup data to the remote region, which facilitates data safety and faster recovery. After adding a remote region as a backup data store, you can't turn it off.
```

File: tidb-cloud/backup-and-restore.md [hunk 6]
```diff
@@ -14,25 +14,47 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+    - In **Backup Retention**, configure the minimum backup data retention period.
+
+    - In **Backup Storage Region**, select the regions where you want to store your backup data. TiDB Cloud stores your backup data in your local region by default. In addition, you can add another remote region, and TiDB Cloud will copy all new backup data to the remote region, which facilitates data safety and faster recovery. After adding a remote region as a backup data store, you can't turn it off.
 
 4. Click **Confirm**.
 
-If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
+> **Note that:**
+> You can not disable automatic backup.
+> To use the PITR feature, firstly make sure your TiDB cluster version is at least v6.3.0 and the TiKV node configuration is at least 8c/16g, then file a ticket to request to enable the PITR feature.
```
Comment (IANTHEREAL) [inline tidb-cloud/backup-and-restore.md:43]:
```suggestion
```

File: tidb-cloud/backup-and-restore.md [hunk 7]
```diff
@@ -14,25 +14,47 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+> After adding a remote region as a backup data store, you can't turn it off.
```
Comment (IANTHEREAL) [inline tidb-cloud/backup-and-restore.md:44]:
```suggestion
```

**Comment by IANTHEREAL** [comment]
already modified, PTAL @ran-huang

File: tidb-cloud/backup-and-restore.md [hunk 8]
```diff
@@ -83,18 +103,26 @@ To restore your TiDB cluster data from a backup to a new cluster, take the follo
 
 1. Navigate to the **Backup** tab of a cluster.
 
-2. Select an existing backup in the list, and click **Restore**.
+2. Click **Restore**. The setting window displays.
+
+3. Select **Restore Mode**, you can
+
+    - In **Select Backup Name**, select an existing backup in the backup list.
+    - If PITR is enabled, you can click **Select Time Point**, select the point in time you want to recover.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:111]:
```suggestion
    <SimpleTab>
    <div label="Select Time Point">

    To restore the new cluster to any point in time, take the following steps:

    1. Click **Select Time Point**.
    2. Select **Date** and **Time** you want to restore to.
    
    </div>

    <div label="Select Backup Name">

    To restore the new cluster to a selected backup, take the following steps:

    1. Click **Select Backup Name**.
    2. Select a backup you want to restore to.

    </div>
```

File: tidb-cloud/backup-and-restore.md [hunk 9]
```diff
@@ -14,25 +14,45 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
 
 4. Click **Confirm**.
 
-If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
+> **Note that:**
+> You can not disable automatic backup.
+
+### Backup storage region support
+
+TiDB Cloud does not yet support adding an arbitrary remote region as backup data storage, the regions already supported are as follows:
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:46]:
```suggestion
Currently, you cannot select an arbitrary remote region for backup data storage. The regions already supported are as follows:
```

File: tidb-cloud/backup-and-restore.md [hunk 10]
```diff
@@ -14,25 +14,45 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
 
-Note that you can not disable automatic backup.
+| Cloud provider | Region                      | Remote regions support   |
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:48]:
```suggestion
| Cloud provider | Local region                      | Remote regions support   |
```

File: tidb-cloud/backup-and-restore.md [hunk 11]
```diff
@@ -14,25 +14,45 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+|----------------|-----------------------------|--------------------------|
+| GCP            | Tokyo (asia-northeast1)     | Osaka (asia-northeast2)  |
+
+> **Note that:**
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:52]:
```suggestion
> **Note:**
>
```

File: tidb-cloud/backup-and-restore.md [hunk 12]
```diff
@@ -14,25 +14,45 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+> TiDB Cloud will charge you for multiple copies of backup storage based on Backup Storage Region setting, and backup storage price varies by region.
+> If you select multiple backup storage regions then you will also incur backup data replication charges.
+> See [Data Backup Cost](https://en.pingcap.com/tidb-cloud-pricing-details/#data-backup-cost) for more information.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:55]:
```suggestion
> Based on the Backup Storage Region setting, you might be charged for multiple copies of backup storage. If you select multiple backup storage regions, you are also charged for backup data replication. The pricing varies by region. For more information, see [Data Backup Cost](https://en.pingcap.com/tidb-cloud-pricing-details/#data-backup-cost).
```

File: tidb-cloud/backup-and-restore.md [hunk 13]
```diff
@@ -83,18 +103,26 @@ To restore your TiDB cluster data from a backup to a new cluster, take the follo
...
-2. Select an existing backup in the list, and click **Restore**.
+2. Click **Restore**. The setting window displays.
+
+3. Select **Restore Mode**, you can
+
+    - In **Select Backup Name**, select an existing backup in the backup list.
+    - If PITR is enabled, you can click **Select Time Point**, select the point in time you want to recover.
 
-3. In the **Restore** window, make the following changes if necessary:
+4. In **Restore to Region**, you can select the same region as the **Backup Storage Region** configured in the **Backup Settings**.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:113]:
```suggestion
4. In **Restore to Region**, you can select the same region as the **Backup Storage Region** configured in the **Backup Settings**.
```

File: tidb-cloud/backup-and-restore.md [hunk 14]
```diff
@@ -113,3 +141,5 @@ To restore a deleted cluster from recycle bin, take the following steps:
    The cluster restore process starts and the **Security Settings** dialog box is displayed.
 
 6. In the **Security Settings** dialog box, set the root password and allowed IP addresses to connect to your cluster, and then click **Apply**.
+
+> **Note that:** Do PITRin the Recycle Bin is not supported.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:145]: 这个是什么意思？

Comment (IANTHEREAL) [inline tidb-cloud/backup-and-restore.md:145]:
```suggestion
> **Note that:**  Recycle Bin doesn't support PITR.
```

File: tidb-cloud/backup-and-restore.md (hunk 10 above)
Comment (IANTHEREAL) [reply tidb-cloud/backup-and-restore.md:48]: how about `cluster region`?

Comment (ran-huang) [reply tidb-cloud/backup-and-restore.md:55]: LGTM~ Updated.

File: tidb-cloud/backup-and-restore.md [hunk 15]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
 
 1. Navigate to the **Backup** tab of a cluster.
 
-2. Click **Auto Setting**. The setting window displays.
+2. Click **Backup Settings**. The setting window displays.
 
-3. In the setting window, select **Backup Time** and maximum backup files in **Limit (backup files)**.
+3. In the setting window, configure the automatic backup:
+
+    - Check whether the PITR(**Point-in-time Recovery**) feature is on.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:31]:
```suggestion
    - Check whether the PITR (**Point-in-time Recovery**) feature is on. 
```

File: tidb-cloud/backup-and-restore.md [hunk 16]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
 
-2. Click **Auto Setting**. The setting window displays.
+2. Click **Backup Settings**. The setting window displays.
 
-3. In the setting window, select **Backup Time** and maximum backup files in **Limit (backup files)**.
+3. In the setting window, configure the automatic backup:
+
+    - Check whether the PITR(**Point-in-time Recovery**) feature is on. 
+
+        To use the PITR feature, make sure your TiDB cluster version is at least v6.3.0 and the TiKV node configuration is at least 8C/16G, and file a ticket to request to enable the PITR feature.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:33]: *Node size* is used on the TiDB Cloud console. Let's use the same term.
```suggestion
        To use the PITR feature, make sure your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8C/16G, and file a ticket to request to enable the PITR feature.
```

File: tidb-cloud/backup-and-restore.md [hunk 17]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+
+    - In **Backup Time**, schedule a start time for the daily cluster backup. 
+
+        It is recommended to schedule automatic backup at the low workload period. If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
+
+    - In **Backup Retention**, configure the minimum backup data retention period.
+
+    - In **Backup Storage Region**, select the regions where you want to store your backup data. 
+
+        TiDB Cloud stores your backup data in your local region by default. In addition, you can add another remote region, and TiDB Cloud will copy all new backup data to the remote region, which facilitates data safety and faster recovery. After adding a remote region as a backup data storage, you cannot turn it off.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:43]:
```suggestion
        TiDB Cloud stores your backup data in the current region of your cluster by default. In addition, you can add another remote region, and TiDB Cloud will copy all new backup data to the remote region, which facilitates data safety and faster recovery. After adding a remote region as a backup data storage, you cannot remove it.
```

> After adding a remote region as a backup data storage, you cannot turn it off.

这句的意思是说添加了 remote region 之后，无法停止 cross region backup，还是指无法删除 remote region 里的备份？

File: tidb-cloud/backup-and-restore.md [hunk 18]
```diff
@@ -113,3 +163,5 @@ To restore a deleted cluster from recycle bin, take the following steps:
    The cluster restore process starts and the **Security Settings** dialog box is displayed.
 
 6. In the **Security Settings** dialog box, set the root password and allowed IP addresses to connect to your cluster, and then click **Apply**.
+
+> **Note that:** Recycle Bin doesn't support PITR.
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:167]:
```suggestion
> **Note:** 
> 
> You cannot restore a deleted cluster to any point in time. You can only select a time point that an automatic or manual backup is performed at.
```

File: tidb-cloud/backup-and-restore.md [hunk 19]
```diff
@@ -113,3 +163,7 @@ To restore a deleted cluster from recycle bin, take the following steps:
    The cluster restore process starts and the **Security Settings** dialog box is displayed.
 
 6. In the **Security Settings** dialog box, set the root password and allowed IP addresses to connect to your cluster, and then click **Apply**.
+
+> **Note:** 
+> 
+> You cannot restore a deleted cluster to any point in time. You can only select an automatic or manual backup  to restore.
```
Comment (IANTHEREAL) [inline tidb-cloud/backup-and-restore.md:169]:
```suggestion
> You cannot restore a deleted cluster to any point in time. You can only select an automatic or manual backup to restore.
```

**Review by ran-huang — APPROVED** [review]

File: tidb-cloud/backup-and-restore.md [hunk 20]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
 
-2. Click **Auto Setting**. The setting window displays.
+2. Click **Backup Settings**. The setting window displays.
 
-3. In the setting window, select **Backup Time** and maximum backup files in **Limit (backup files)**.
+3. In the setting window, configure the automatic backup:
+
+    - Check whether the PITR (**Point-in-time Recovery**) feature is on. 
+
+        To use the PITR feature, make sure your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8C/16G, and file a ticket to request to enable the PITR feature.
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:33]:
```suggestion
        To use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8C/16G, and file a ticket to request the PITR feature.
```

File: tidb-cloud/backup-and-restore.md [hunk 21]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+
+    - In **Backup Time**, schedule a start time for the daily cluster backup. 
+
+        It is recommended to schedule automatic backup at the low workload period. If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:37]:
```suggestion
        It is recommended to schedule automatic backup at a low workload period. If you do not specify a preferred backup time, TiDB Cloud assigns a default backup time, which is 2:00 AM in the time zone of the region where the cluster is located.
```

File: tidb-cloud/backup-and-restore.md [hunk 22]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+
+    - In **Backup Retention**, configure the minimum backup data retention period.
+
+    - In **Backup Storage Region**, select the regions where you want to store your backup data. 
+
+        TiDB Cloud stores your backup data in the current region of your cluster by default. In addition, you can add another remote region, and TiDB Cloud will copy all new backup data to the remote region, which facilitates data safety and faster recovery. After adding a remote region as a backup data storage, you cannot remove it.
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:43]:
```suggestion
        TiDB Cloud stores your backup data in the current region of your cluster by default. In addition, you can add another remote region, and TiDB Cloud will copy all new backup data to the remote region, which facilitates data safety and faster recovery. After adding a remote region as a backup data storage, you cannot remove the region.
```

File: tidb-cloud/backup-and-restore.md [hunk 23]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+> **Note:**
+>
+> You can not disable automatic backup.
+
+### Backup storage region support
 
-Note that you can not disable automatic backup.
+Currently, you cannot select an arbitrary remote region for backup data storage. The regions already supported are as follows:
+
+| Cloud provider | Custer region                      | Remote regions support   |
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:55]:
```suggestion
| Cloud provider | Custer region                      | Remote region support   |
```

File: tidb-cloud/backup-and-restore.md [hunk 24]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+|----------------|-----------------------------|--------------------------|
+| GCP            | Tokyo (asia-northeast1)     | Osaka (asia-northeast2)  |
+
+> **Note:**
+>
+> Based on the Backup Storage Region setting, you might be charged for multiple copies of backup storage. If you select multiple backup storage regions, you are also charged for backup data replication. The pricing varies by region. For more information, see [Data Backup Cost](https://en.pingcap.com/tidb-cloud-pricing-details/#data-backup-cost).
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:61]:
```suggestion
> Based on the Backup Storage Region setting, you might be charged for multiple replicas of backup storage. If you select multiple backup storage regions, you are also charged for backup data replication. The pricing varies by region. For more information, see [Data Backup Cost](https://en.pingcap.com/tidb-cloud-pricing-details/#data-backup-cost).
```

File: tidb-cloud/backup-and-restore.md [hunk 25]
```diff
@@ -83,18 +109,42 @@ To restore your TiDB cluster data from a backup to a new cluster, take the follo
...
 
-2. Select an existing backup in the list, and click **Restore**.
+2. Click **Restore**. The setting window displays.
+
+3. In **Restore Mode**, you can choose to restore the new cluster to any point in time or to a selected backup.
+
+    <SimpleTab>
+    <div label="Select Time Point">
+
+    To restore the new cluster to any point in time, take the following steps:
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:119]:
```suggestion
    To restore data of any point in time to a new cluster, take the following steps:
```

File: tidb-cloud/backup-and-restore.md [hunk 26]
```diff
@@ -83,18 +109,42 @@ To restore your TiDB cluster data from a backup to a new cluster, take the follo
...
+
+    1. Click **Select Time Point**.
+    2. Select **Date** and **Time** you want to restore to.
+    
+    </div>
+
+    <div label="Select Backup Name">
 
-3. In the **Restore** window, make the following changes if necessary:
+    To restore the new cluster to a selected backup, take the following steps:
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:128]:
```suggestion
    To restore a selected backup to the new cluster, take the following steps:
```

Comment (TomShawn) [reply tidb-cloud/backup-and-restore.md:128]: LGTM

File: tidb-cloud/backup-and-restore.md [hunk 27]
```diff
@@ -83,18 +109,42 @@ To restore your TiDB cluster data from a backup to a new cluster, take the follo
 
 1. Navigate to the **Backup** tab of a cluster.
 
-2. Select an existing backup in the list, and click **Restore**.
+2. Click **Restore**. The setting window displays.
+
+3. In **Restore Mode**, you can choose to restore the new cluster to any point in time or to a selected backup.
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:114]:
```suggestion
3. In **Restore Mode**, you can choose to restore data of any point in time or a selected backup to a new cluster.
```

File: tidb-cloud/backup-and-restore.md [hunk 28]
```diff
@@ -83,18 +109,42 @@ To restore your TiDB cluster data from a backup to a new cluster, take the follo
...
 
-3. In the **Restore** window, make the following changes if necessary:
+    To restore the new cluster to a selected backup, take the following steps:
 
+    1. Click **Select Backup Name**.
+    2. Select a backup you want to restore to.
+
+    </div>
+
+4. In **Restore to Region**, you can select the same region as the **Backup Storage Region** configured in the **Backup Settings**.
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:135]:
```suggestion
4. In **Restore to Region**, select the same region as the **Backup Storage Region** configured in the **Backup Settings**.
```

File: tidb-cloud/backup-and-restore.md [hunk 29]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
 
-2. Click **Auto Setting**. The setting window displays.
+2. Click **Backup Settings**. The setting window displays.
 
-3. In the setting window, select **Backup Time** and maximum backup files in **Limit (backup files)**.
+3. In the setting window, configure the automatic backup:
+
+    - Check whether the PITR (**Point-in-time Recovery**) feature is on. 
+
+        To use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8C/16G, and file a ticket to request the PITR feature.
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:33]:
```suggestion
        To use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8 vCPU and 16 GiB, and file a ticket to request the PITR feature.
```

File: tidb-cloud/backup-and-restore.md [hunk 30]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
 
 1. Navigate to the **Backup** tab of a cluster.
 
-2. Click **Auto Setting**. The setting window displays.
+2. Click **Backup Settings**. The setting window displays.
 
-3. In the setting window, select **Backup Time** and maximum backup files in **Limit (backup files)**.
+3. In the setting window, configure the automatic backup:
+
+    - Check whether the PITR (**Point-in-time Recovery**) feature is on.
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:31]: PITR 对于自动 backup 是必须的吗，还是一个可选项呢？改动前的自动 backup 都是不需要 PITR 的。是否可以简单介绍些 PITR 的使用场景？

File: tidb-cloud/backup-and-restore.md [hunk 31]
```diff
@@ -83,18 +109,42 @@ To restore your TiDB cluster data from a backup to a new cluster, take the follo
 
 1. Navigate to the **Backup** tab of a cluster.
 
-2. Select an existing backup in the list, and click **Restore**.
+2. Click **Restore**. The setting window displays.
+
+3. In **Restore Mode**, you can choose to restore data of any point in time or a selected backup to a new cluster.
+
+    <SimpleTab>
+    <div label="Select Time Point">
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:117]: 这里是否需要提一下，要 Select Time Point，需要确保备份时已经开启 PITR 呢？

File: tidb-cloud/backup-and-restore.md [hunk 32]
```diff
@@ -83,18 +109,42 @@ To restore your TiDB cluster data from a backup to a new cluster, take the follo
...
+
+    To restore data of any point in time to a new cluster, take the following steps:
```
Comment (ran-huang) [inline tidb-cloud/backup-and-restore.md:119]:
```suggestion
    To restore data of any point in time to a new cluster, make sure that **PITR** in **Backup Settings** is on and then take the following steps:
```

Comment (TomShawn) [reply tidb-cloud/backup-and-restore.md:119]: LGTM

File: tidb-cloud/backup-and-restore.md (hunk 30 above)
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:31]:
```suggestion
    - (Optional) Check whether the PITR (**Point-in-time Recovery**) feature is on. 
```

File: tidb-cloud/backup-and-restore.md [hunk 33]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
 1. Navigate to the **Backup** tab of a cluster.
 
-2. Click **Auto Setting**. The setting window displays.
+2. Click **Backup Settings**. The setting window displays.
 
-3. In the setting window, select **Backup Time** and maximum backup files in **Limit (backup files)**.
+3. In the setting window, configure the automatic backup:
+
+    - Check whether the PITR (**Point-in-time Recovery**) feature is on. 
+
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:32]:
```suggestion

    PITR supports restoring data of any point in time to a new cluster. You can use it to:

    - Reduce RPO in disaster recovery.
    - Resolve cases of data write errors by restoring point-in-time that is before the error event.
    - Audit the historical data of the business.

```

File: tidb-cloud/backup-and-restore.md [hunk 34]
```diff
@@ -14,25 +14,51 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+        To use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8 vCPU and 16 GiB. Currently, PITR is in beta. To enable it, [file a ticket](/tidb-cloud/tidb-cloud-support.md) to request the PITR feature.
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:33]:
```suggestion
        If you have one of the preceding needs and want to use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8 vCPU and 16 GiB. Currently, PITR is in beta. To enable it, [file a ticket](/tidb-cloud/tidb-cloud-support.md) to request the PITR feature.
```

File: tidb-cloud/backup-and-restore.md [hunk 35]
```diff
@@ -14,25 +14,57 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+
+Currently, you cannot select an arbitrary remote region for backup data storage. The regions already supported are as follows:
+
+| Cloud provider | Custer region                      | Remote region support   |
+|----------------|-----------------------------|--------------------------|
+| GCP            | Tokyo (asia-northeast1)     | Osaka (asia-northeast2)  |
+
+> **Note:**
+>
+> Based on the Backup Storage Region setting, you might be charged for multiple replicas of backup storage. If you select multiple backup storage regions, you are also charged for backup data replication. The pricing varies by region. For more information, see [Data Backup Cost](https://en.pingcap.com/tidb-cloud-pricing-details/#data-backup-cost).
```
[the remark this answers is not in the document]
Comment (ran-huang) [reply tidb-cloud/backup-and-restore.md:67]: 改了一点。那个 “OUT” 是什么意思

[the remark this answers is not in the document]
Comment (IANTHEREAL) [reply tidb-cloud/backup-and-restore.md:67]: 代表收取出流量的费用。 借鉴 s3 的文档，For CRR, you also pay for inter-region Data Transfer OUT from S3 to each destination region. https://aws.amazon.com/s3/pricing/

[the remark this answers is not in the document]
Comment (ran-huang) [reply tidb-cloud/backup-and-restore.md:67]: 改成小写的 out 了，其他 lgtm

File: tidb-cloud/backup-and-restore.md [hunk 36]
```diff
@@ -14,25 +14,57 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
 
 1. Navigate to the **Backup** tab of a cluster.
 
-2. Click **Auto Setting**. The setting window displays.
+2. Click **Backup Settings**. The setting window displays.
 
-3. In the setting window, select **Backup Time** and maximum backup files in **Limit (backup files)**.
+3. In the setting window, configure the automatic backup:
+
+    - (Optional) Check whether the PITR (**Point-in-time Recovery**) feature is on.
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:31]:
```suggestion
    - (Optional) Enable the PITR (**Point-in-time Recovery**) feature according to your needs. 
```

File: tidb-cloud/backup-and-restore.md [hunk 37]
```diff
@@ -14,25 +14,57 @@ This document describes how to back up and restore your TiDB cluster data on TiD
...
+
+    - (Optional) Check whether the PITR (**Point-in-time Recovery**) feature is on. 
+
+        PITR supports restoring data of any point in time to a new cluster. You can use it to:
+
+        - Reduce RPO in disaster recovery.
+        - Resolve cases of data write errors by restoring point-in-time that is before the error event.
+        - Audit the historical data of the business.
+
+        If you have one of the preceding needs and want to use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8 vCPU and 16 GiB. Currently, PITR is in beta. To enable it, [file a ticket](/tidb-cloud/tidb-cloud-support.md) to request the PITR feature.
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:39]:
```suggestion
        If you have one of the preceding needs and want to use the PITR feature, make sure that your TiDB cluster version is at least v6.3.0 and the TiKV node size is at least 8 vCPU and 16 GiB. Currently, PITR is in beta. To enable it, [file a ticket](/tidb-cloud/tidb-cloud-support.md).
```

File: tidb-cloud/backup-and-restore.md (hunk 36 above)
Comment (IANTHEREAL) [reply tidb-cloud/backup-and-restore.md:31]: No switch is implemented, only a status whether it is on or not.

File: tidb-cloud/backup-and-restore.md [hunk 38]
```diff
@@ -83,18 +115,42 @@ To restore your TiDB cluster data from a backup to a new cluster, take the follo
...
 
-2. Select an existing backup in the list, and click **Restore**.
+2. Click **Restore**. The setting window displays.
+
+3. In **Restore Mode**, you can choose to restore data of any point in time or a selected backup to a new cluster.
+
+    <SimpleTab>
+    <div label="Select Time Point">
+
+    To restore data of any point in time to a new cluster, make sure that **PITR** in **Backup Settings** is on and then take the following steps:
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:125]:
```suggestion
    To restore data of any point in time to a new cluster, make sure that you have enabled [**PITR**](#automatic-backup) in **Backup Settings** for the backup and then take the following steps:
```

**Review by TomShawn — APPROVED** [review]
LGTM

File: tidb-cloud/backup-and-restore.md [hunk 39]
```diff
@@ -83,18 +115,42 @@ To restore your TiDB cluster data from a backup to a new cluster, take the follo
...
 
-3. In the **Restore** window, make the following changes if necessary:
+    <div label="Select Backup Name">
 
+    To restore a selected backup to the new cluster, take the following steps:
+
+    1. Click **Select Backup Name**.
+    2. Select a backup you want to restore to.
+
+    </div>
```
Comment (TomShawn) [inline tidb-cloud/backup-and-restore.md:139]:
```suggestion
    </div>
    </SimpleTab>
```

**Pull request merged**